charts/astronomer/values.yaml

```diff
@@ -22,25 +22,21 @@ images:
     tag: 3.11.5-4
     pullPolicy: IfNotPresent
   houston:
-    repository: astronomerinc/ap-houston-api
-    tag: 0.19.9
+    repository: asoldatenko/ap-houston-api
```

Comment:
Are we shipping code that references non-astronomer repositories?

Reply:
I think this is what @andriisoldatenko is trying to get at here based on the slack comment, its a special case different image to test from the houston feature branch that contains this. Because of how things are wired up, In order for us to test from one of our docker registries the prisma2 changes have to be merged to main branch on houston. Which we don't want yet because we want to test. 

Feel free to correct me if I'm wrong here andrii

Reply:
Although, I would think we should be able to push to our registry without going through CI. Maybe that isn't true.

Reply:
We definitely could push this to our own registry. I don't know if it makes that much of a difference though, and this might actually be better since it won't be exposed to folks browsing our docker hub tags (unlikely, but possible.)